hw5/ChangeArray.java

```diff
@@ -0,0 +1,24 @@
+package hw4;
+
+import java.util.Arrays;
+
+public class ChangeArray {
```

Comment:
Looks like you do not check the last element in array. So, your last element always will has default value. In such a case 0.

Reply:
deleted -1 from "i < array.length-1"